test/unit/forge/db/models/TeamType_spec.js

```diff
@@ -1,6 +1,9 @@
 const should = require('should') // eslint-disable-line
 const setup = require('../setup')
 
+let objectCount = 0
+const generateName = (root = 'object') => `${root}-${objectCount++}`
```

Comment:
The `TeamType` sequelize model doesn't have 'unique: true' set on the name field. This means, for a clean install, the constraint doesn't get applied and the tests pass.

However, the migration that was added to introduce TeamType *does* set the unique constraint - so that is what we have in production.

I chose to match production with out baseline - so the constraint is applied. So I had to update the tests to use unique names below.